examples/shared_samples/hypervisor_manager.rb

```diff
@@ -22,6 +22,10 @@
 # api_version = 800 & variant = C7000 to OneviewSDK::API800::C7000::HypervisorManager
 # api_version = 800 & variant = Synergy to OneviewSDK::API800::Synergy::HypervisorManager
```

Comment:
Add API1000, API1200 for synergy and c7000 under supportedAPI's section and in the section present below it.

Reply:
@chebroluharika Changes Updated